### PR TITLE
Update faq

### PR DESCRIPTION
**Pull request by blacksponge**
## Purpose

Update FAQ.

## Context

Based on #434 

## Changes

Change some wording in the FAQ.

## How to test this PR

\-

### CODE REVIEW

**Comment by blacksponge** [comment]
Rebased on top of develop, please review.

**Comment by matsduf** [comment]
It says "update of FAQ" but most files changed are the translations files for the strings.

**Comment by matsduf** [comment]
It looks like all translations of the FAQ are gone. Is that correct?

File: src/app/components/faq/faq.component.html [hunk 1]
```diff
@@ -58,32 +59,36 @@ <h2 i18n="faq section header">General information</h2>
...
-    </p>
-    <p i18n="faq answer">
-      Lastly, this open source version of Zonemaster was built using modular code which basically means that you can integrate parts of it in your own systems, if you wish. For example, it is quite rare that you would want a complete program just to check for redelegations.
-    </p>
+    <ul>
+      <li i18n="faq answer">
+        Zonemaster saves all history from earlier tests based on the tested domain name, which means you can go back to a test you did some time ago and compare it to the test you ran just a moment ago.
+      </li>
+      <li i18n="faq answer">
+        All tests that Zonemaster runs are defined in the Test Case Specification documents that can be found in the <a href="https://github.com/zonemaster/zonemaster/tree/master/docs/public/specifications/tests#zonemaster-test-case-specifications">Test Case Specification documents</a>
```
[the remark this answers is not in the document]
Comment (blacksponge) [reply src/app/components/faq/faq.component.html]: fixed

File: src/app/components/faq/faq.component.html [hunk 2]
```diff
@@ -58,32 +59,36 @@ <h2 i18n="faq section header">General information</h2>
...
+      The judgement of Zonemaster is primarily based on the DNS standards as defined in <a i18n-href="RFC Wikipedia link" href="https://en.wikipedia.org/wiki/Request_for_Comments">RFCs</a>. It also bases its judgement on DNS best practices, which can be more loosely defined.
     </p>
 
+    <p i18n="faq answer">
+      All Zonemaster tests are defined in the <a href="https://github.com/zonemaster/zonemaster/tree/master/docs/public/specifications/tests#zonemaster-test-case-specifications">Test Case Specification documents</a> in which the references to the standard documents for each test case are found.
+    </p>
     <p i18n="faq answer">
       The descriptions of message levels such as <i>notice</i>, <i>warning</i> and <i>error</i> are found in <a href="https://github.com/zonemaster/zonemaster/blob/master/docs/public/specifications/tests/SeverityLevelDefinitions.md">Severity Level Definitions</a>.
```
[the remark this answers is not in the document]
Comment (blacksponge) [reply src/app/components/faq/faq.component.html]: done

**Pull request merged**